sys-apps/policycoreutils/policycoreutils-3.6-r1.ebuild

```diff
@@ -0,0 +1,210 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_{10..11} )
+PYTHON_REQ_USE="xml(+)"
+
+inherit python-r1 toolchain-funcs bash-completion-r1
+
+MY_PV="${PV//_/-}"
+MY_P="${PN}-${MY_PV}"
+EXTRAS_VER="1.37"
+
+DESCRIPTION="SELinux core utilities"
+HOMEPAGE="https://github.com/SELinuxProject/selinux/wiki"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/SELinuxProject/selinux.git"
+	SRC_URI="!vanilla? ( https://dev.gentoo.org/~perfinion/distfiles/policycoreutils-extra-${EXTRAS_VER}.tar.bz2 )"
+	S1="${WORKDIR}/${P}/${PN}"
+	S2="${WORKDIR}/policycoreutils-extra"
+	S="${S1}"
+else
+	SRC_URI="https://github.com/SELinuxProject/selinux/releases/download/${MY_PV}/${MY_P}.tar.gz
+		!vanilla? ( https://dev.gentoo.org/~perfinion/distfiles/policycoreutils-extra-${EXTRAS_VER}.tar.bz2 )"
+	KEYWORDS="~amd64 ~arm ~arm64 ~mips ~x86"
+	S1="${WORKDIR}/${MY_P}"
+	S2="${WORKDIR}/policycoreutils-extra"
+	S="${S1}"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="audit pam +python split-usr vanilla"
+REQUIRED_USE="
+	!vanilla? ( python ${PYTHON_REQUIRED_USE} )
+"
+
+DEPEND="
+	python? (
+		>=sys-libs/libselinux-${PV}:=[python,${PYTHON_USEDEP}]
+		>=sys-libs/libsemanage-${PV}:=[python(+),${PYTHON_USEDEP}]
+		audit? ( >=sys-process/audit-1.5.1[python,${PYTHON_USEDEP}] )
+		${PYTHON_DEPS}
+	)
+	!python? (
+		>=sys-libs/libselinux-${PV}:=
+		>=sys-libs/libsemanage-${PV}:=
+		audit? ( >=sys-process/audit-1.5.1 )
+	)
+	>=sys-libs/libsepol-${PV}:=
+	sys-libs/libcap-ng:=
+	pam? ( sys-libs/pam:= )
+	!vanilla? (
+		>=app-admin/setools-4.2.0[${PYTHON_USEDEP}]
+	)
+"
+
+# Avoid dependency loop in the cross-compile case, bug #755173
+# (Still exists in native)
+BDEPEND="sys-devel/gettext"
+
+# pax-utils for scanelf used by rlpkg
+RDEPEND="${DEPEND}
+	app-misc/pax-utils"
+
+PDEPEND="sys-apps/semodule-utils
+	python? ( sys-apps/selinux-python )"
+
+src_unpack() {
+	# Override default one because we need the SRC_URI ones even in case of 9999 ebuilds
+	default
+	if [[ ${PV} == 9999 ]] ; then
+		git-r3_src_unpack
+	fi
+}
+
+src_prepare() {
+	S="${S1}"
+	cd "${S}" || die "Failed to switch to ${S}"
+	if [[ ${PV} != 9999 ]] ; then
+		# If needed for live ebuilds please use /etc/portage/patches
+		eapply "${FILESDIR}/policycoreutils-3.1-0001-newrole-not-suid.patch"
+	fi
+
+	if ! use vanilla; then
+		# rlpkg is more useful than fixfiles
```

Comment:
oh, do you actually want `fixfiles` instead of rlpkg? or was this just cuz rlpkg is removed for vanilla? Cuz last i looked (many many years ago to be fair), fixfiles was really redhat-specific and didnt make a ton of sense on gentoo. I'll need to look again but it should either be unconditionally added or unconditionally never there. Its such a tiny binary that (assuming it works fine on gentoo) there is no harm in always having it.